plugin/inlay_hint.py

```diff
@@ -142,12 +145,21 @@ def format_inlay_hint_label(inlay_hint: InlayHint, session: Session, phantom_uui
             })
             result += f'<a href="{inlay_hint_click_command}">'
         instruction_text = '\nDouble-click to insert' if has_text_edits else ""
-        result += f'<span title="{(tooltip + instruction_text).strip()}">{html.escape(label)}</span>'
+        truncated = len(label) > truncate_limit
+        truncated_label = label[:truncate_limit] + '…' if truncated else label
+        result_tooltip = (tooltip + instruction_text).strip()
+        result += f'<span title="{result_tooltip}">{html.escape(truncated_label)}</span>'
         if is_clickable:
             result += "</a>"
+        if truncated:
+            truncation_tooltip = html.escape('\n' + label)
+            result = f"<span title=\"{truncation_tooltip}\">{result}</span>"
         return result
 
+    remaining_truncate_limit = truncate_limit
     for label_part in label:
+        if remaining_truncate_limit < 0:
```

Comment:
```suggestion
        if remaining_truncate_limit <= 0:
```

Reply:
We need to change 

`truncated = len(raw_label) > remaining_truncate_limit`

 to
 
`truncated = len(raw_label) >= remaining_truncate_limit`

Otherwise, if the inlay hint length matches the truncate limit, no dots are applied (limit is 10 characters):

![possible-bug](https://github.com/user-attachments/assets/8abad664-4c1f-4d09-8715-c0deea24dbe7)

Reply:
Done in [abe0987](https://github.com/sublimelsp/LSP/pull/2514/commits/abe09875b6b058e45fd97fda86b92df0ab838881)